src/views/AddItem.jsx

```diff
@@ -11,13 +11,20 @@ export function AddItem({ listToken }) {
 		isSuccess: false,
 	};
 
+	const itemNames = new Set(
+		data.map((item) => item.name.toLowerCase().replace(/\s+/g, '')),
+	);
+
 	const [form, setForm] = useState(defaultFormValues);
 
 	const handleSubmit = async (event) => {
 		event.preventDefault();
-		if (form.itemName.length === 0) {
+
+		if (itemNames.has(form.itemName.toLowerCase().replace(/\s+/g, ''))) {
```

Comment:
We should also check if `form.itemName` is an empty string, because it still is possible to add an empty item by adding a bunch of spaces

Reply:
Thanks for pointing that out! I've added a check for this now.